test/verify/check-kdump

```diff
@@ -419,11 +418,10 @@ class TestKdumpNFS(KdumpHelpers):
         self.machines["nfs"].write("/etc/exports", "/srv/kdump 10.111.113.0/24(rw,no_root_squash)\n")
         self.machines["nfs"].execute("mkdir -p /srv/kdump/var/crash; firewall-cmd --add-service nfs; systemctl restart nfs-server")
 
+        # there shouldn't be any crash reports in the target directory
+        self.assertEqual(m.execute("find /var/crash -maxdepth 1 -mindepth 1 -type d"), "")
+
         # set up client machine
-        if m.image.startswith("fedora"):
-            m.execute("systemctl disable kdump")
```

Comment:
Ah, that addresses my earlier comment. Please reverse the commit order or squash.